uxarray/remap/inverse_distance_weighted.py

```diff
@@ -0,0 +1,272 @@
+from __future__ import annotations
+from typing import TYPE_CHECKING, Union
+
+if TYPE_CHECKING:
+    from uxarray.core.dataset import UxDataset
+    from uxarray.core.dataarray import UxDataArray
+
+import numpy as np
+
+import uxarray.core.dataarray
+import uxarray.core.dataset
+from uxarray.grid import Grid
+import warnings
+
+
+def _inverse_distance_weighted_remap(
+    source_grid,
+    destination_grid,
+    source_data,
+    remap_to="nodes",
+    coord_type="spherical",
+    power=2,
+    k_neighbors=8,
```

Comment:
k_neighbors=8 is on the higher side. Would it make sense to lower it?

Reply:
8 is the default value from what I found, also 8 isn't that high when most meshes have high numbers of nodes. Also, the point is each neighbor is weighted differently, so as you move further from the destination point each neighbor is weighted as less "important" so to speak in the final calculation of the destination data point.

Reply:
Yeah 8 seems reasonable, as long as we allow the user to customize it and keep it documented, it's all good.